recipes/strands-agents/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set python_min = "3.10" %}
+{% set name = "strands-agents" %}
+{% set version = "0.1.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/strands_agents-{{ version }}.tar.gz
+  sha256: 667c6d706167f74e762ab652174d9267c39e213a94c7e7dced1dbc42e7513cd8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - hatchling
+    - hatch-vcs
+    - pip
+  run:
+    - python {{ python_min }}
```

See https://github.com/conda-forge/staged-recipes/pull/30353#issuecomment-2981952797

```suggestion
    - python >={{ python_min }}
```